BootloaderCorePkg/Tools/CommonUtility.py

```diff
@@ -37,28 +37,36 @@ class LZ_HEADER(Structure):
 # Hash values defined should match with cryptolib.h
 HASH_TYPE_VALUE = {
 # {   Hash_string:        Hash_Value}
-            "SHA2_256"    : 0,
-            "SHA2_384"    : 1,
-            "SHA2_512"    : 2,
-            "SM3_256"     : 3,
+            "SHA2_256"    : 1,
+            "SHA2_384"    : 2,
+            "SHA2_512"    : 3,
+            "SM3_256"     : 4,
     }
 
 # Sign values defined should match with cryptolib.h
 SIGN_TYPE_VALUE = {
 # {   Sign_string:        Sign_Value}
-            "RSA2048"     : 0,
-            "RSA3072 "    : 1,
-            "SM2     "    : 2,
+            "RSA2048"     : 1,
+            "RSA3072 "    : 2,

```

Comment:
I think we discussed this earlier and decided that in the new data structures, a signature will be identified with 3 properites: 
sig_size (2K, 3K, 4K), key_type(RSA, ECC, etc) and sig_alg(PKCSv1.5, PSS etc).

What is the purpose of SIGN_TYPE_VALUE?

Reply:
Yes, This won't get used. I would clean up this.